src/advanced/configuration-updater.md

```diff
@@ -128,8 +128,8 @@ config][config_propose] serialization. It has the following fields:
   String containing JSON serialization of proposed configuration.
 - **votes_history_hash**: Hash  
   Hash of the proposed configuration.
-- **num_votes**: integer  
-  Number of votes for the proposed configuration.
+- **num_validators**: integer  
+  Number of available validators.
```

Comment:
"Available validators" isn't very descriptive wording... Do I understand correctly that `num_validators` is the number of validators that can vote for the proposal (or, in other words, the maximum number of possible votes for the proposal)? If so, propose to elaborate using these words.

Reply:
Will "Number of validators that can vote for the proposal" be fine?

Reply:
Yeah 👍 .